store/store.go

```diff
@@ -231,30 +240,44 @@ func NewUbuntuStoreSnapRepository(cfg *SnapUbuntuStoreConfig, storeID string) *S
 		cfg = &defaultConfig
 	}
 
-	fields := cfg.DetailFields
-	if fields == nil {
-		fields = detailFields
+	fieldList := cfg.DetailFields
+	if fieldList == nil {
+		fieldList = detailFields
 	}
 
+	fields := strings.Join(fieldList, ",")
```

Comment:
Ah! good point. Should I do directly, i.e.
`fields := url.Values{"fields": []string{strings.Join(fieldList, ",")}}.Encode()`?